ias-lib/library/ias.entity.lua

```diff
@@ -798,6 +806,12 @@ end
 
 ---
 ---@env SERVER
+---@param vars table
+---* called when a template is loaded that contains `vars` key
+---
+function Entity:onTemplateVariables(vars)

```

Comment:
prob should be called `onTemplateVariablesLoad` since you have `onTemplateVariablesSave`